url_test.go

```diff
@@ -15,7 +18,7 @@ func TestURLParser(t *testing.T) {
 	} {
 		t.Run(path, func(t *testing.T) {
 			t.Skip("test uses network, sometimes fails for no reason")
-			got, err := readmeURL(path)
+			got, err := readmeURL(context.Background(), path)
```

Comment:
This maybe as you are on Go 1.24

```suggestion
			got, err := readmeURL(t.Context(), path)
```